blacklist/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
Would it not make sense to create a constant for that and use it in the dozen of places below?

Reply:
With this being part of the gradle wrapper I wanted to keep it pretty standard. I'm not even sure whether you can import constants?

Reply:
Especially with all the samples projects being essentially seperate

Reply:
Yes, agree - even though idea is good the change seems quite involved.